### PR TITLE
October 2017

### PR DESCRIPTION
**Pull request by jcblw**
here is the new event @rouzbeh84 do not have your main info let me know when you get it, but I think that is the last bit before we announce.

### CODE REVIEW

**Comment by rouzbeh84** [comment]
k, sounds good @jcblw. I should have a title and description tonight/tomorrow at the latest. Want me to post here or just make a commit on your PR?

**Comment by ashwell** [comment]
I'll start writing the email tomorrow, if everything is set, I'd say we can announce Tuesday.

**Comment by jcblw** [comment]
@rouzbeh84 of course you just adding the commit would be nicer, but either works for me.

@ashwell thanks man. Lets shoot for Tuesday.

File: public/_data.json [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
-            "image": "kurtis-rainbolt-greene.png",
-            "twitter": "krainboltgreene",
-            "github": "krainboltgreene",
-            "description": "Babel, how it's progressed, the future of web development, and what it solves today."
+            "title": "hotsauce.js",
+            "name": "Rouzbeh Sarrafieh",
+            "image": "rouzbeh-sarrafieh.jpg",
+            "twitter": "rouzbeh84",
+            "github": "rouzbeh84",
+            "description": "spice up my life"
```
Comment (ashwell) [inline public/_data.json]: 😂

**Pull request merged**